tests/test_computed_fields.py

```diff
@@ -62,6 +62,13 @@ def double_width(self) -> int:
     assert rect.model_dump() == {'width': 10, 'length': 5, 'area': 50, 'area2': 50}
     assert rect.model_dump_json() == '{"width":10,"length":5,"area":50,"area2":50}'
 
+    assert set(Rectangle.model_fields) == {'width', 'length'}
+    assert set(Rectangle.model_computed_fields) == {'area', 'area2'}
+
+    assert Rectangle.model_computed_fields['area'].description == 'An awesome area'
+    assert Rectangle.model_computed_fields['area2'].title == 'Pikarea'
+    assert Rectangle.model_computed_fields['area2'].description == 'Another area'
```

Comment:
I would also check what happens when trying to set the `model_computed_fields` attribute to some value (for reference, having the chain of `@classmethod` + `@property` was deprecated in 3.11 partly because of unsupported behavior when setting values on those properties).

Reply:
Is it supposed to be settable ?